src/main/java/core/basesyntax/Area.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Area {

```

Comment:
Don't use verbs for class/interface names. Consider renaming the interface to something that represents an entity or a property rather than an action, like `AreaCalculator`.